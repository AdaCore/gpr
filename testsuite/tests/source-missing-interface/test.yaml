description:
    Test for source update
driver: build_and_run
project_file: source_missing_interface
main: main
