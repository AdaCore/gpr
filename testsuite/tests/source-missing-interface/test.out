Messages found:
demo.gpr:8:38: source for interface unit 'toto' not found
